### ISSUE TITLE
flow from dataframe and flow methods provide different validation loss plots

### ISSUE THREAD

**Issue by pooraniR** [keras-team/keras#13950]
I tried to run a couple of pretrained architectures with the same configuration using both flow and flow_from_dataframe methods. 

There are differences in the outputs. Could someone explain why that's happening?

The most concerning difference that I noticed is the increased volatility in the validation loss plot when flow_from_dataframe method is used. By volatility, I'm referring to high fluctuations in the validation loss.

**Comment by Saduf2019** [comment]
@pooraniR 
Moving this issue to closed status as there has been no activity, in case you still face the error please create a new issue.
